app/src/main/res/values/attrs.xml

```diff
@@ -34,6 +34,7 @@
         <attr name="min_lines" format="integer" />
         <attr name="background_alpha" format="integer" />
         <attr name="multiline" format="boolean" />
+        <attr name="placeholder" format="string" />
```

Comment:
**Suggestion**: You should override `<attr name="android:hint" />`

Reply:
why do I want to do this ?